tests/Stripe/Service/ApplePayDomainServiceTest.php

```diff
@@ -23,7 +23,7 @@ final class ApplePayDomainServiceTest extends \PHPUnit\Framework\TestCase
      */
     protected function setUpService()
     {
-        $this->client = new \Stripe\StripeClient('sk_test_123', null, MOCK_URL);
+        $this->client = new \Stripe\StripeClient(['api_key' => 'sk_test_123', 'api_base' => MOCK_URL]);
```

Comment:
Minor but as I see this change, can we make this a global variable? If we ever want to add something or change the default key or whatever it would make the PR more readable

Reply:
I plan on cleaning up the service tests later by factorizing code in a `ServiceTestHelper` trait or something like that.